namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-contacts-preprod/resources/irsa.tf

```diff
@@ -0,0 +1,49 @@
+# Add the names of the SQS queues & SNS topics which the app needs permissions to access.
+# The value of each item should be the namespace where the queue or topic was created.
+# This information is used to collect the IAM policies which are used by the IRSA module.
+locals {
+  # The names of the SNS topics used and the namespace which created them
+  sns_topics = {
+    "cloud-platform-Digital-Prison-Services-e29fb030a51b3576dd645aa5e460e573" = "hmpps-domain-events-preprod"
+  }
+
```

Comment:
I think we're missing the audit queue definitions here? `sqs_queues = {
    "Digital-Prison-Services-preprod-hmpps_audit_queue" = "hmpps-audit-preprod",
  }`